src/main/groovy/cz/alenkacz/gradle/jsonvalidator/ValidateJsonSchemaSyntaxTask.groovy

```diff
@@ -0,0 +1,44 @@
+package cz.alenkacz.gradle.jsonvalidator
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.github.fge.jackson.JacksonUtils
+import com.github.fge.jsonschema.cfg.ValidationConfiguration
+import com.github.fge.jsonschema.processors.syntax.SyntaxValidator
+import groovy.io.FileType
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+
```

Comment:
I would be really nice if you could add some class level documentation here so it will help developer community if to understand this code and contribute.